Source/Charts/Renderers/LegendRenderer.swift

```diff
@@ -431,6 +431,10 @@ open class LegendRenderer: Renderer
                 
                 var posX = originPosX
                 
+                if let labelColor = e.labelColor {
```

Comment:
this should be `let labelTextColor = e.labelColor ?? legendTextColor`

Reply:
@jjatie It's been a long time. I have to double-check it